app/src/main/java/mollie/tictactoe/GameHelper.java

```diff
@@ -0,0 +1,58 @@
+package mollie.tictactoe;
+
+import mollie.tictactoe.players.MobilePlayer;
+import ttt.game.Board;
+import ttt.game.GameEngine;
+import ttt.game.Marks;
+
+public class GameHelper {
+
+    private GameEngine mGameEngine;
```

Comment:
@gemcfadyen this is Android convention.  All member variables should begin with an `m` and I thougt it best to be consistent across all classes, whether plain Java or not.